docs/local_resource.md

```diff
@@ -57,3 +57,30 @@ For more on trigger mode, [see the docs](https://docs.tilt.dev/manual_update_con
 `auto_init=False` is currently only compatible with `TRIGGER_MODE_MANUAL`. If
 you'd like a local resource that runs automatically in response to file changes
 but does NOT run on `tilt up`, [let us know](https://tilt.dev/contact).
+
+## serve_cmd
+
+local_resource's `serve_cmd` argument allows a local_resource to function as a
+persistent process, so you can use it for things like running services locally
+instead of in k8s, or `tail -f`.
+
+Without `serve_cmd`, a local_resource functions as a sort of batch job - when
+Tilt runs it, it expects it to terminate. While it's running, it's "in progress",
```

Comment:
maybe "Tilt runs the command and expects it to terminate. While the command is running, the resource is 'in progress'..."